.github/workflows/npm-publish.yml

```diff
@@ -1,8 +1,8 @@
 name: npm-publish
 on:
   push:
-    branches:
-      - 'master'
+    tags:
+      - 'v*.*.*'
```

Comment:
Can we remove the [`if condition`](https://github.com/hitz-group/dynamo-helper/blob/9bd60085da08bf5475f3f1d71237b1e154cba538/.github/workflows/npm-publish.yml#L11) as we are moving to tag based approach ?
```
if: contains( github.event.head_commit.message, 'Release***')
```

Would need the same fixes in [`github action`](https://github.com/hitz-group/dynamo-helper/blob/master/.github/workflows/github-publish.yml#L5-L13) too.